project/src/js/ui/tournament/detail/socketHandler.js

```diff
@@ -0,0 +1,23 @@
+import * as xhr from '../tournamentXhr';
+import throttle from 'lodash/function/throttle';
+import m from 'mithril';
+
+export default function(ctrl) {
+  let reload = throttle((c) => {
+    xhr.reload(c.tournament().id).then(c.reload);
+  }, 2000);
+
+  const handlers = {
+    reload: function() {
+      reload(ctrl);
+    },
+    resync: function() {
```

Comment:
Ok, I will ask @ornicar how I should now deal with the socket version and resync.